Essentials/src/config.yml

```diff
@@ -785,10 +785,10 @@ chat:
   # Chat formatting can be done in two ways, you can either define a standard format for all chat.
   # Or you can give a group specific chat format, to give some extra variation.
   # For more information of chat formatting, check out the wiki: http://wiki.ess3.net/wiki/Chat_Formatting
+  # Note: Using the {PREFIX} and {SUFFIX} placeholders along with {DISPLAYNAME} may cause double prefixes/suffixes to be shown in chat
```

Comment:
This should probably mention under *what* scenario `{PREFIX}` and `{SUFFIX}` will result in duplication, namely when `add-prefix-suffix: true` is set.